opa/views/res_partner_views.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="inherit_res_partner_form_view" model="ir.ui.view">
```

Comment:
Use the same ID as original view, that would prevent inheriting the same view several times

```suggestion
    <record id="view_partner_form" model="ir.ui.view">
```